### PR TITLE
Windows installation: remove dependencies [10490]

### PR DESCRIPTION
**Pull request by richiware**
Described dependencies for Windows binaries installation are not necessary. This PR removes them.

### CODE REVIEW

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Please fix DCO

File: docs/installation/binaries/binaries_windows.rst [hunk 1]
```diff
@@ -39,60 +38,6 @@ If Visual Studio is already installed but the Visual C++ Redistributable package
...
-The latest OpenSSL version for Windows can be found in the `OpenSSL website <https://www.openssl.org/>`_.
-After installing, add the environment variable :code:`OPENSSL_ROOT_DIR` pointing to the installation root directory.
-
-For example:
-
-.. code-block:: bash
-
-   OPENSSL_ROOT_DIR=C:\Program Files\OpenSSL-Win64
-
-.. _install_bw:
```
Comment (jparisu) [inline docs/installation/binaries/binaries_windows.rst:94]: Do not remove this tag

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/installation/binaries/binaries_windows.rst (hunk 1 above)
Comment (richiware) [reply docs/installation/binaries/binaries_windows.rst:94]: Done

**Review by jparisu — DISMISSED** [review]
LGTM
Please fix DCO

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/737/)

**Review by rsanchez15 — CHANGES_REQUESTED** [review]
Sign last commit

**Comment by richiware** [comment]
Build status: [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Docs%20Manual/741/)

**Review by rsanchez15 — APPROVED** [review]
LGTM

**Pull request merged**